egs/libricss/SURT/prepare.sh

```diff
@@ -118,9 +126,12 @@ fi
 
 if [ $stage -le 4 ] && [ $stop_stage -ge 4 ]; then
   log "Stage 4: Extract features for LibriSpeech, trim to alignments, and shuffle the cuts"
-  python local/compute_fbank_librispeech.py
-  lhotse combine data/manifests/librispeech_cuts_train* - |\
-    lhotse cut trim-to-alignments --type word --max-pause 0.2 - - |\
+  # python local/compute_fbank_librispeech.py
+  lhotse combine data/manifests/librispeech_cuts_train* data/manifests/librispeech_cuts_train_all.jsonl.gz
+  lhotse cut trim-to-alignments --type word --max-pause 0.2 \
+    data/manifests/librispeech_cuts_train_all.jsonl.gz \
+    data/manifests/librispeech_cuts_train_all_trimmed.jsonl.gz
+  cat <(gunzip -c data/manifests/librispeech_cuts_train_all_trimmed.jsonl.gz) | \
```

Comment:
I think the original command works fine without having to save all the intermediate manifests?

Reply:
@JinZr is this change needed?

Reply:
Yes i suppose. The old version does not work on my side.

Reply:
Hmm, weird. Well, anyway, it's a minor issue.